src/functions-reference/higher-order_functions.Rmd

```diff
@@ -10,6 +10,7 @@ if (knitr::is_html_output()) {
 cat(' * <a href="functions-algebraic-solver.html">Algebraic Equation Solver</a>\n')
 cat(' * <a href="functions-ode-solver.html">Ordinary Differential Equation (ODE) Solvers</a>\n')
 cat(' * <a href="functions-1d-integrator.html">1D Integrator</a>\n')
+cat(' * <a href="functions-reduce.html">Reduce-Sum</a>\n')
 cat(' * <a href="functions-map.html">Higher-Order Map</a>\n')
```

Comment:
note that title here is "Higher-Order Map" but subsection name is "Map-Rect" - change here or?